### PR TITLE
API-12128 limit event object to request fields only in sendEvent methods v3.3

### PR DESCRIPTION
**Pull request by wfabjanczuk**
Jira: https://livechatinc.atlassian.net/browse/API-12128

### CODE REVIEW

File: src/agent/rtm.ts [hunk 1]
```diff
@@ -233,10 +233,10 @@ export default class RTM extends RTMAPI {
    * It's possible to write to a chat without joining it. The user sending an event will be automatically added to the chat
    * with the present parameter set to false.
    * @param chat_id - chat to send event to
-   * @param event - Event object
+   * @param event - Event object limited to request fields only
```
Comment (litwinow) [inline src/agent/rtm.ts]: OPT: `Event request object`

File: src/customer/web.ts [hunk 2]
```diff
@@ -91,10 +91,10 @@ export default class Web extends WebAPI {
    * It's possible to write to a chat without joining it. The user sending an event will be automatically added to the chat
    * with the present parameter set to false.
    * @param chat_id - chat to send event to
-   * @param event - Event object
+   * @param event - Event object limited to request fields only
```
Comment (litwinow) [inline src/customer/web.ts]: as above

**Review by litwinow — APPROVED** [review]

**Review by maciejjwojcik — APPROVED** [review]

File: src/agent/rtm.ts (hunk 1 above)
Comment (wfabjanczuk) [reply src/agent/rtm.ts]: fixed

File: src/customer/web.ts (hunk 2 above)
Comment (wfabjanczuk) [reply src/customer/web.ts]: fixed

**Pull request merged**